tpm2/tpm2.go

```diff
@@ -794,6 +794,17 @@ func PolicyPCR(rw io.ReadWriter, session tpmutil.Handle, expectedDigest []byte,
 	return err
 }
 
+// PolicyOr compares PolicySession→Digest against the list of provided values.
+// If the current Session→Digest does not match any value in the list,
+// the TPM shall return TPM_RC_VALUE. Otherwise, the TPM will reset policySession→Digest
+// to a Zero Digest. Then policySession→Digest is extended by the concatenation of
+// TPM_CC_PolicyOR and the concatenation of all of the digests.
+func PolicyOr(rw io.ReadWriter, session tpmutil.Handle, digests TPMLDigest) error {
```

Comment:
For my reference, whats the intended use of PolicyOr? like what kind of funky policy invariants does it let you do?

Reply:
Ask that people who invented Intel TXT ;)